tools/clang/test/HLSL/cpp-errors.hlsl

```diff
@@ -572,8 +567,8 @@ void expressions()
   internal->fn();                 // expected-error {{operator is not supported}}
   local_i = (int3) { 1, 2, 3 };   // expected-error {{compound literal is unsupported in HLSL}}
 
-  // `class` ok, but component count should be checked earlier (1 to 4 uniform components):
-  Texture2D<::c_outer_fn> local_texture;
+  // ok, but component count should be checked earlier (1 to 4 uniform components):
+  Texture2D<::c_outer_td> local_texture;
```

Comment:
Comment seems out of place now, see previous comment about the same change.

Reply:
Removed in latest commit